chat/src/helpers/hybrid_query.py

```diff
@@ -47,7 +43,7 @@ def hybrid_query(query: str, model_id: str, vector_field: str = "embedding", k:
                     "normalization-processor": {
                         "combination": {
                             "parameters": {
-                                "weights": weights
+                                "weights": [0.25, 0.75]
```

Comment:
More of a question, but these also have to be updated in Meadow terraform, right?

Reply:
In Meadow's [`Meadow.Search.Index.create_search_pipeline/4`](https://github.com/nulib/meadow/blob/deploy/staging/app/lib/meadow/search/index.ex#L51), yes.